OOP/InputAction.java

```diff
@@ -0,0 +1,165 @@
+import java.util.Scanner;
+
+
+public class InputAction implements Action{
+static private Train.Type type;
+	public void  perform(int input)
+	{
+		
+	if(input==1)
+	{
+		
+		type=Train.Type.PASSENGER;
+		
+		
+	}else
+	{
+		
+		type=Train.Type.GOODS;
+		
+	}
+		
+	}
+	
+	public static void InputProcess() {
```

Comment:
ACTION: This method better suits in the where it was previously as it involves major calls like fare-calulations etc. BUT make sure no menu-related SOPs are there.